app/utils/NPMRepositoryParser.scala

```diff
@@ -18,7 +18,10 @@ object NPMRepositoryParser extends AbstractParser {
     // project files
     val buildFiles = getBuildFiles(folder)
 
-    val dependencies = buildFiles.map(getDependencies(_, folder)).reduce((d1, d2) => d1 ++ d2)
+    val dependencies = buildFiles
+      .map(getDependencies(_, folder))
+      .reduceOption((d1, d2) => d1 ++ d2)
```

Comment:
I'm ok with this one 😄 You can also use a `fold`, i don't know if it's better 
```java
    .fold(Seq.empty[Dependency])((d1, d2) => d1 ++ d2)
```

Reply:
I prefer the Java-like `reduceOption/getOrElse`, but I changed it as you suggested 😛

Reply:
If you want a more scala like change you can write it like that 😄 
```java
    .fold(Seq.empty[Dependency])(_++_)
```
i'm not too fond of this solution